apps/filecoin-site/src/app/blog/components/BlogCard.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link'
+
+import {
+  CardImage,
+  type CardImageProps,
+} from '@filecoin-foundation/ui/Card/CardImage'
+
+import { Heading } from '@/components/Heading'
+import { TagGroup } from '@/components/TagGroup/TagGroup'
+
+import type { BlogPost } from '../types/blogPostType'
+
+import { PostMetadata } from './PostMetadata'
+
+type BlogCardLink = {
+  href: string
+  ariaLabel: string
+}
+
+export type BlogCardProps = {
+  title: BlogPost['title']
+  cta?: BlogCardLink
+  tags?: BlogPost['categories']
+  description?: BlogPost['excerpt']
```

Comment:
Is description optional?

Reply:
I made `cta`, `description` and `tags` as not optional 👍